ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.container.js

```diff
@@ -110,13 +110,16 @@ import { subtractHexes } from '../../../../shared/modules/conversion.utils';
 import ConfirmTransactionBase from './confirm-transaction-base.component';
 
 let customNonceValue = '';
-const customNonceMerge = (txData) =>
-  customNonceValue
+const customNonceMerge = (txData) => {
+  console.log('after customNonceMerge', { ...txData, customNonceValue });
```

Comment:
Should we remove this `console.log`? same for line 243.

Reply:
Good spot, done.